lms/templates/vert_module.html

```diff
@@ -17,3 +17,20 @@ <h2 class="hd hd-2 unit-title">${unit_title}</h2>
   %endif
 % endfor
 </div>
+
+<script>
+  $(function(){
+    var ext = '.csv';
+    $('a[href*="' + ext + '"]').each(function() {
```

Comment:
this will select anchors matching the given query so we can be sure that the `href` value will not be empty and `if` can be avoided.
plus you should change to 
```
$('a[href*="' + ext + '"]').each(function(anchor) {}
```
now you can access `anchor.href` directly.

Reply:
by using .each(function(anchor), anchor.href is not accessible.